diff --git a/src/3rdparty/chromium/base/containers/checked_iterators.h b/src/3rdparty/chromium/base/containers/checked_iterators.h
index 0c98b8f..e2f91ac 100644
--- a/src/3rdparty/chromium/base/containers/checked_iterators.h
+++ b/src/3rdparty/chromium/base/containers/checked_iterators.h
@@ -225,7 +225,7 @@ using CheckedContiguousConstIterator = CheckedContiguousIterator<const T>;
 }  // namespace base
 
 #if defined(_LIBCPP_VERSION) && !BUILDFLAG(IS_NACL)
-// Specialize both std::__is_cpp17_contiguous_iterator and std::pointer_traits
+// Specialize both std::__libcpp_is_contiguous_iterator and std::pointer_traits
 // for CCI in case we compile with libc++ outside of NaCl. The former is
 // required to enable certain algorithm optimizations (e.g. std::copy can be a
 // simple std::memmove under certain circumstances), and is a precursor to
@@ -245,7 +245,7 @@ using CheckedContiguousConstIterator = CheckedContiguousIterator<const T>;
 namespace std {
 
 template <typename T>
-struct __is_cpp17_contiguous_iterator<::base::CheckedContiguousIterator<T>>
+struct __libcpp_is_contiguous_iterator<::base::CheckedContiguousIterator<T>>
     : true_type {};
 
 template <typename T>
